maintenance/RESTORE_BACKUPS.md

```diff
@@ -0,0 +1,15 @@
+# How to Restore a backup hosted on an external openshift instance
+
+**NOTE: if you are simply trying to restore a backup from the same openshift instance, skip directly to step 4**
+**NOTE: if performing a restore on a live instance of PIMS, see maintenance page documentation [here](./README.md)**
+
+0. Optionally, enable the display of the maintenance page if the restore is targeting a live instance of PIMS.
+1. login to the openshift environment hosting the DB you wish to copy the backup from.
+1. change to the project hosting the desired database using: oc project \${projectName}
```

Comment:
Would be nice to use code Markdown for any CLI commands.
`oc project \${projectName}`